### DISCUSSION TITLE
为代码中的模型提供商排序

### DISCUSSION THREAD

**Discussion by BrandonStudio** [lobehub/lobehub#3411] [Ideas]
目前各个代码中
- `src/app/api/chat/agentRuntime.ts`
- `src/components/ModelProviderIcon/index.tsx`
- `src/config/llm.ts`
- `src/config/modelProviders/index.ts`
- `src/server/globalConfig/index.ts`
- `src/store/user/slices/modelList/action.ts`

模型提供商的配置是按照时间排序的，新加的提供商排在最后。这就导致，同时进行的PR必定产生冲突。

为此，提议将这类文件中模型提供商的配置按照数字字母 0-9 aA-zZ 排序，极大地减少PR产生的冲突

**Comment by arvinxx** [comment]
new ai provider 已经支持自定义排序
